staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/conversion.go

```diff
@@ -26,69 +26,10 @@ import (
 	"k8s.io/apimachinery/pkg/conversion"
 	"k8s.io/apimachinery/pkg/fields"
 	"k8s.io/apimachinery/pkg/labels"
-	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
-func AddConversionFuncs(scheme *runtime.Scheme) error {
```

Comment:
Where did these go?

Reply:
They are now autogenerated by zz_generated.conversions.go:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/zz_generated.conversion.go#L88

and registered in AddToGroupVersion:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/register.go#L76